internal/pkg/postprocessor/extractor/object_storage_azure.go

```diff
@@ -0,0 +1,80 @@
+package extractor
+
+import (
+	"encoding/xml"
+	"fmt"
+	"strings"
+
+	"github.com/internetarchive/Zeno/pkg/models"
+)
+
+// Azure Blob Storage
+var azureServers = []string{
+	// Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0
+	"Windows-Azure-Blob",
+	// Blob Service Version 1.0 Microsoft-HTTPAPI/2.0
+	"Blob Service Version",
+	// emulator, https://github.com/Azure/Azurite
+	"Azurite-Blob",
+}
```

Comment:
I have tested Zeno with the azure extracor locally with [Azurite](https://github.com/Azure/Azurite/) and it works. But I haven't run it on a real Azure Blob yet.

Also, I found that some Azure Blobs use the `Server: Blob Service Version XXX...` header, and I don't know what the difference is between them and `Server: Windows-Azure-Blob...`.